### PR TITLE
update to patternfly v5

### PR DESCRIPTION
**Pull request by InsaneZein**
### Description
Update to PFv5

[RHCLOUD-31884](https://issues.redhat.com/browse/RHCLOUD-31884)

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [x] Change reviewed for extraneous code 
- [x] UI best practices adhered to 
- [ ] Commits squashed and meaningfully named 
- [x] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [ ] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

**Comment by InsaneZein** [comment]
Konflux marked the unit test step as a failure even though all the tests passed :thinking:

**Comment by fhlavac** [comment]
@InsaneZein looks like a timeout perhaps this would help https://tekton.dev/docs/pipelines/pipelineruns/#configuring-a-failure-timeout

**Comment by fhlavac** [comment]
Just one oddity I found in the search input

Before:
image

Now:
image

**Comment by InsaneZein** [comment]
Oh I didn't catch that. Thanks, Filip!

File: .cache_vbpib4
Comment (fhlavac) [inline .cache_vbpib4:1]: this can be removed

**Comment by fhlavac** [comment]
@InsaneZein if I am not mistaken, @karelhala is taking a look at the unit tests separately - https://github.com/RedHatInsights/user-preferences-frontend/pull/205#issuecomment-2373419052 

So we may be good to merge this one

File: .tekton/user-preferences-frontend-pull-request.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ metadata:
   name: user-preferences-frontend-on-pull-request
   namespace: rh-platform-experien-tenant
 spec:
+  timeouts:
+    pipeline: "2h0m0s"
```
Comment (fhlavac) [inline .tekton/user-preferences-frontend-pull-request.yaml:20]:
```suggestion
```

**Pull request merged**